packages/faustwp-core/package.json

```diff
@@ -26,6 +36,7 @@
     "fetch-mock": "9.11.0",
     "jest": "^27.3.1",
     "rimraf": "^3.0.2",
+    "shx": "^0.3.4",
```

Comment:
[shx](https://www.npmjs.com/package/shx) is a safe way to use cp across different operating systems.